### ISSUE TITLE
Slide height maybe should be bigger

### ISSUE THREAD

**Issue by rodriciru** [PrestaShop/theme-refacto#127]
https://github.com/PrestaShop/theme-refacto/blob/4148dcdc9c572d177ea0e4b20f6fb290ae59b619/src/scss/partials/variables/prestashop/_carousel.scss#L14

340px is too low for products, but its great for home slider
![imagen](https://user-images.githubusercontent.com/10741347/144216279-f78a040f-ee3f-44bc-adf1-919486e21cdf.png)
 whe can put anything here so i will stuck to 557.5px (which is the width of the image in the bigger container), or better 100% which as prestashop makes fixed sized images on products (800x800), theres no be any problems.

2 options: 
1. Add a new height for home slide, or product slide
OR
2. Use BS5 ratio clases for home slide https://getbootstrap.com/docs/5.1/helpers/ratio/

**Comment by SharakPL** [comment]
Yeah, this again 🤦‍♂️ There should not be any height set at all. Already [tried](https://github.com/PrestaShop/PrestaShop/issues/26373#issuecomment-949826004) to report this on Classic theme, but some of the maintainers consider this behaviour (cutting of part of the slides) OK 😕 Slides should be fully responsive and operational on mobile.

**Comment by rodriciru** [comment]
the problem here was that i write a height thinking in home slider, and break product slider.
I will PR whit changes so both sides are fine IMO
